CollAction/Controllers/ProjectsController.cs

```diff
@@ -412,13 +413,31 @@ public ViewResult StartInfo()
 
             if (success)
             {
+                var projUrl = this.Url.Action("Details", "Projects", new { name=commitProjectViewModel.ProjectName}, HttpContext.Request.Scheme); 
+                var systemUrl = $"{HttpContext.Request.Scheme}://{HttpContext.Request.Host}{HttpContext.Request.PathBase}";
+                var userDescription = String.IsNullOrEmpty(user?.FirstName) ? "" : user.FirstName;
```

Comment:
Null coalescing operator would be a bit nicer here:
user?.FirstName ?? "";

Reply:
fixed